--- conftest.py
+++ conftest.py
@@ -15,6 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.

+import os
 import socket
 import subprocess
 import time
@@ -25,6 +26,7 @@ from _pytest.skipping import MarkEvaluator

 from selenium import webdriver
 from selenium.webdriver import DesiredCapabilities
+from selenium.webdriver.common.utils import free_port
 from test.selenium.webdriver.common.webserver import SimpleWebServer
 from test.selenium.webdriver.common.network import get_lan_ip

@@ -76,6 +78,19 @@ def driver(request):
                 yield
                 return

+    if driver_class == 'Chrome':
+        port = free_port()
+        capabilities = DesiredCapabilities.CHROME.copy()
+
+        if os.environ.get('WITH_DEVTOOLS_PROXY') == 'true':
+            capabilities['chromeOptions'] = {
+                'binary': os.environ['CHROME_WRAPPER_PATH'],
+                'args': [
+                    '--devtools-proxy-binary={}'.format(os.environ['DEVTOOLS_PROXY_PATH']),
+                    '--devtools-proxy-chrome-debugging-port={}'.format(port),
+                ]
+            }
+        kwargs['desired_capabilities'] = capabilities
     if driver_class == 'BlackBerry':
         kwargs.update({'device_password': 'password'})
     if driver_class == 'Firefox':
